src/CTA.WebForms/FileConverters/CodeFileConverter.cs

```diff
@@ -80,17 +80,27 @@ public override async Task<IEnumerable<FileInformation>> MigrateFileAsync()
             {
                 await allMigrationTasks;
             }
-            // NOTE: We use Exception here instead of AggregateException as sometimes await
-            // will auto un-wrap aggregate exception
-            catch (Exception e)
+            // NOTE: We don't provide a reference to the caught exception here because any
+            // encountered exceptions will be logged when examining the individual failed tasks
+            catch
```

Comment:
If an exception gets thrown inside the try, will it immediately go to the catch and then stop processing the other tasks in the WhenAll? Or will it try to process all the tasks no matter what? If that case, can this catch be run many times for 1 try?

A test might help me see what is going on here if it's not too hard to write.

Reply:
`Task.WhenAll` creates a single task that doesn't complete until it runs each of the tasks passed to it to completion regardless if one or more error out before some of the others are finished. It wraps all errors encountered into a single `AggregateException` which is passed to the `catch` block (the `catch` block only needs to be triggered once this way). One thing to note is that the original exceptions can still be accessed via the original tasks that created them, which is how we are currently accessing them in the `catch` block